src/framework/uicomponents/view/popupwindow/popupwindow_qquickview.cpp

```diff
@@ -63,6 +63,7 @@ void PopupWindow_QQuickView::init(QQmlEngine* engine, bool isDialogMode, bool is
     // dialog
     if (isDialogMode) {
         m_view->setFlags(Qt::Dialog);
+        m_view->setIcon(QIcon(uiConfiguration()->appIconPath().toString()));
```

Comment:
Wouldn't it be safer to set it app-wide? https://doc.qt.io/qt-6/qguiapplication.html#windowIcon-prop
Or does that not work?

Reply:
Yes, this is of course preferable, I tried QGuiApplication::setWindowIcon, but it didn't work